README.md

```diff
@@ -1,5 +1,13 @@
-# The Control File for the openSUSE Language Addon Product #
+Obsoleted Module
+================
 
-[![Travis Build](https://travis-ci.org/yast/skelcd-control-openSUSE-LangAddOn.svg?branch=master)](https://travis-ci.org/yast/skelcd-control-openSUSE-LangAddOn)
-[![Jenkins Build](http://img.shields.io/jenkins/s/https/ci.opensuse.org/yast-skelcd-control-openSUSE-LangAddOn-master.svg)](https://ci.opensuse.org/view/Yast/job/yast-skelcd-control-openSUSE-LangAddOn-master/)
+**This module is obsoleted and is not developed anymore.**
 
+The Language Add On is no longer created by OpenSUSE.
+Instead it uses online repositories.
+
+
+GIT
+---
+
+If you want to see the last state check the [*master_old*](../master_old) branch.
```

Comment:
I noticed that all `master_old` links do not work anymore :worried: See e.g. https://github.com/yast/yast-bluetooth module.

I found that the currently working path is `../../tree/master_old`.

We should probably fix all the dropped packages... :grimacing: